### ISSUE TITLE
Does `<if type="aaa bbb ccc">` without `match="any"` or `match="none"` make sense?

### ISSUE THREAD

**Issue by customcommander** [citation-style-language/styles#3405]
Hi,

Quick question:

I have noticed that some styles (around 85 of them) have multiple values in a `type` attribute without a `match="any"` or `match="none"`:

```xml
<if type="thesis manuscript personal_communication speech motion_picture">
```
Cf https://github.com/citation-style-language/styles/blob/master/ens-de-lyon-centre-d-ingenierie-documentaire.csl#L521

Would that ever evaluate to true though? My understanding is that the default `match` value is `all` and a reference can only have one type. (Cf https://github.com/citation-style-language/schema/blob/master/csl-data.json#L9)

Cheers

**Comment by adam3smith** [comment]
Correct, those would always be false -- we should see if we can add a CI check for this because we clearly aren't catching them in manual QA. Thanks for noticing.
Patches for existing styles welcome.

**Comment by customcommander** [comment]
@adam3smith See #3407 please.
